salt/_states/metalk8s_cordon.py

```diff
@@ -70,4 +69,19 @@ def node_uncordoned(name, **kwargs):
     name
         the name of the node
     '''
+    all_node_taints = __salt__['metalk8s_kubernetes.node_taints'](
+        name, **kwargs
+    )
+    key_to_remove = "node.kubernetes.io/unschedulable"
```

Comment:
Add a comment to say why we remove this taint
Something like
```
k-c-m 1.12 add taint `node.kubernetes.io/unschedulable` when cordonning a node but k-c-m 1.11 does not remove this one when uncordonning
```

Reply:
yeah I can add a comment within the codebase... But the commit message already has this